java/src/main/java/ai/onnxruntime/OrtProvider.java

```diff
@@ -23,7 +23,8 @@ public enum OrtProvider {
   ARM_NN("ArmNNExecutionProvider"),
   ROCM("ROCMExecutionProvider"),
   CORE_ML("CoreMLExecutionProvider"),
-  XNNPACK("XnnpackExecutionProvider");
+  XNNPACK("XnnpackExecutionProvider"),
+  AZURE("AzureExecutionProvider");
```

Comment:
There needs to be an `addAzure` method or a similar method to the C# `AppendExecutionProvider` method if you want to enable that EP in Java. At the moment we have `addXnnpack` which puts the string in itself, but it could be refactored to expose a generic one and make `addXnnpack` and `addAzure` call it.

Reply:
Thanks!
Add the EP here just to pass the nuget packaging pipeline, not to make it available in java.